content/en/docs/tutorials/kubernetes-basics/update/update-intro.html

```diff
@@ -29,8 +29,7 @@ <h3>Updating an application</h3>
 
             <p>Users expect applications to be available all the time and developers are expected to deploy new versions of them several times a day. In Kubernetes this is done with rolling updates. <b>Rolling updates</b> allow Deployments' update to take place with zero downtime by incrementally updating Pods instances with new ones. The new Pods will be scheduled on Nodes with available resources.</p>
 
-            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable during the update and the maximum number of new Pods that can be created, is one. Both options can be configured to either numbers or percentages (of Pods).
-            In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
+            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable and can be created during the update, is one. Both options can be configured to either a fixed quantity or percentile of Pods. In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
```

Comment:
This change loses something. How about:
```suggestion
            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable during the update is one, and the number of temporary extra Pods that can be created during the update is also one. Both options can be configured to either a fixed quantity or percentile of Pods. In Kubernetes, updates are versioned and any Deployment update can be reverted to a previous (stable) version.</p>
```